### DISCUSSION TITLE
如果英文或 ReST 语法标记前面或后面为标点符号，是否要空格？

### DISCUSSION THREAD

**Discussion by 180909** [greyli/flask-docs-zh#88] [Q&A] [answered]
如果英文前面或后面为标点符号，是否要空格？

**Comment by greyli** [comment, marked as answer]
不需要。对应的原文可以贴出来看一下。

**Reply by 180909** [reply]
比如说：` Flask， `还是` Flask ， `
前者英文后无空格，后者英文后有空格。

**Reply by greyli** [reply]
这种不需要空格。

**Comment by yuxiaoy1** [comment]
英文前面为中文标点符号，不需要加空格；英文前面为英文标点符号，需要加空格。
英文后面的标点符号，不要加空格。

**Comment by frostming** [comment]
这个可以参考pofmt格式化后的效果，与 @yuxiaoy1 相同

**Comment by greyli** [comment]
我想 @180909 指的是译文里的空格。译文里只有中文标点，所以不需要加空格。
